endpoints/server_test.go

```diff
@@ -229,6 +239,10 @@ func TestServerServeHTTP(t *testing.T) {
 		{"POST", "TestDefault", `{"age": 20}`, ``, http.StatusOK},
 		{"POST", "TestDefault", `{"weight": 3.14}`, ``, http.StatusOK},
 		{"POST", "TestDefault", `{"name":"francesc", "age": 20}`, ``, http.StatusOK},
+		{"POST", "TestMinMax", `{"age":10,"weight":5,"grade":"C"}`, ``, http.StatusOK},
+		{"POST", "TestMinMax", `{"age":123,"weight":5,"grade":"C"}`, ``, http.StatusBadRequest},
+		{"POST", "TestMinMax", `{"age":10,"weight":1,"grade":"C"}`, ``, http.StatusBadRequest},
+		{"POST", "TestMinMax", `{"age":10,"weight":5,"grade":"G"}`, ``, http.StatusBadRequest},
```

Comment:
If you change `MinMaxMsg` struct to have `Age` with `endpoints:"min=0,max=100,req` tag (or maybe it's "required", don't remember), and then add this to the tests table:

```go
{"POST", "TestMinMax", `{"age":0,"weight":5,"grade":"C"}`, ``, http.StatusOK}
```

will it pass?